### ISSUE TITLE
Dead link in moderation administration section

### ISSUE THREAD

**Issue by honzie** [coralproject/talk#2772]
**Expected behavior:**

When I click on the link under 'Configuration' in the 'Toxic Comment Filter Section', I expect to learn more about how to set up the Perspective Comment API. The text on the page (en-US) is below:

"For additional information on how to set up the Perspective Toxic Comment Filter please visit: https://github.com/conversationai/perspectiveapi/blob/master/quickstart.md"

My expectation _might_ be to go to https://github.com/conversationai/perspectiveapi/tree/master/1-get-started. However, there may be a better getting started guide—this is the one I could find in the repo.

The code is in `admin.ftl` files, nearly 3 dozen, since they are split across locales:

```
configure-moderation-perspective-accountNote =
  For additional information on how to set up the Perspective Toxic Comment Filter please visit:
  <externalLink>https://github.com/conversationai/perspectiveapi/blob/master/quickstart.md</externalLink>
```

**Actual behavior:**

I get a 404 at: https://github.com/conversationai/perspectiveapi/blob/master/quickstart.md

**Related Issues:** 

I didn't see anything opened that looked similar

**Versions:**

- NodeJS: 12.13.1
- NPM: 6.12.1
- MongoDB: 4.2
- Redis: 3.2
- Browser: Chrome 79
- OS: osX

**Comment by honzie** [comment]
I'd be happy to swap out the links, if someone on the team validates that the above link (https://github.com/conversationai/perspectiveapi/tree/master/1-get-started) is what is expected for Perspective API. There definitely might be a better one.

**Comment by kgardnr** [comment]
Hey @honzie thanks for catching this! We'd love if you could open a PR to swap this out.

Could you actually link is to this: https://github.com/conversationai/perspectiveapi#readme

I think that's more general and less likely to break if they update their docs links in the future.

Thank you!

**Comment by honzie** [comment]
Absolutely! That's a good idea. Probably a better link for sure.

**Comment by kgardnr** [comment]
Fixed in #2773
